.github/workflows/celix_etcdlib.yml

```
@@ -0,0 +1,36 @@
+name: Standalone Celix etcdlib Weekly
+
+on:
+  schedule:
+    - cron:  '0 0 * * 0'
```

We probably want to build this on push & pull_requests as well.
```suggestion
on:
  push:
  pull_request:
  schedule:
    - cron:  '0 0 * * 0'
```

---

I want to prevent to  much builds. So IMO just once a week to ensure etcdlib still builds as separate project is good enough.

Note that etcdlib is build on every push / PR.